meteor/server/api/ingest/generation.ts

```diff
@@ -244,11 +243,6 @@ export async function calculateSegmentsFromIngestData(
 				})
 				res.parts.push(part)
 
-				// This ensures that it doesn't accidently get played while hidden
-				if (blueprintSegment.segment.isHidden) {
-					part.invalid = true
```

Comment:
Why is this being removed? Seems dangerous.

Reply:
This was moved to the commit stage, to allow for the segment to be kept visible if it is still playing
https://github.com/nrkno/tv-automation-server-core/pull/638/files#diff-432a5ee01b84991e0799a0ccb9960dbad910336d7a5f4a5494feb0fb1f971e07R276